File: Bars.GkhGJI/libs/B4/model/BasePlanAction.js
Ext.define('B4.model.BasePlanAction', {
    extend: 'B4.model.InspectionGji',
    idProperty: 'Id',

    proxy: {
        type: 'b4proxy',
        controllerName: 'BasePlanAction'
    },
    fields: [
        { name: 'Municipality', defaultValue: null },
        { name: 'TypeBase', defaultValue: 110 },
        { name: 'Plan', defaultValue: null },
        { name: 'Contragent', defaultValue: null },
        { name: 'SubjectName' },
        { name: 'PersonAddress' },
        { name: 'DateStart' },
        { name: 'DateEnd' },
        { name: 'CountDays' },
        { name: 'Requirement' },
        { name: 'State', defaultValue: null },
        { name: 'MoSettlement' },
        { name: 'PlaceName' }
    ]
});

File: Bars.GkhGJI/Entities/Inspection/BasePlanAction.cs
namespace Bars.GkhGji.Entities
{
    using System;

    /// <summary>
    /// Основание проверки план мероприятий 
    /// </summary>
    public class BasePlanAction : InspectionGji
    {

        /// <summary>
        /// План мероприятий
        /// </summary>
        public virtual PlanActionGji Plan { get; set; }

        /// <summary>
        /// Место нахождения (или Адрес Физ лица)
        /// Данное поле заполняется втом случае если выбрали тип субъект = ФизЛицо 
        /// </summary>
        public virtual string PersonAddress { get; set; }

        /// <summary>
        /// Дата начала
        /// </summary>
        public virtual DateTime? DateStart { get; set; }

        /// <summary>
        /// Дата окончания
        /// </summary>
        public virtual DateTime? DateEnd { get; set; }

        /// <summary>
        /// Рабочих дней
        /// </summary>
        public virtual int? CountDays { get; set; }

        /// <summary>
        /// Требование
        /// </summary>
        public virtual string Requirement { get; set; }

    }
}